ui/ui.go

```diff
@@ -380,16 +403,17 @@ func waitForStatusMessageTimeout(appCtx applicationContext, t *time.Timer) tea.C
 // document. Note that we could be doing things like checking if the file is
 // a directory, but we trust that gitcha has already done that.
 func localFileToMarkdown(cwd string, res gitcha.SearchResult) *markdown {
-	md := &markdown{
+	return &markdown{
 		localPath: res.Path,
 		Note:      stripAbsolutePath(res.Path, cwd),
 		Modtime:   res.Info.ModTime(),
 	}
-
-	return md
 }
 
 func stripAbsolutePath(fullPath, cwd string) string {
+	if cwd == "" {
+		cwd, _ = os.Getwd()
```

Comment:
is ok to ignore the error here right?

Reply:
I guess it's okay to ignore it here, but I perhaps `os.Getwd` shouldn't be called here since this is a helper function

Reply:
yea i agree, it goes out of the strip expected logic

Reply:
I moved `os.Getwd()` to elsewhere.

I also noticed we could just use the stdlib for that helper so I just changed that.